internal/suggest/testdata/test.0005/out.expected

```diff
@@ -1,6 +1 @@
-Found 5 candidates:
```

Comment:
Why does this change?

Reply:
\*Update\* Did some digging into this & seems like we've hit a limitation with `go/scanner` as it doesn't seem to be able to indicate whether the cursor `@` is still inside a comment or not.

eg. [`newTokenIterator`](https://github.com/mdempsky/gocode/blob/083c47a0575d9fd46740b212a7d23274d4f808f0/internal/suggest/cursorcontext.go#L26) returns the same set of tokens for the two example below:
```go
// internal/suggest/testdata/test.0005/test.go.in
package main

...

// methods SetC and SetD are defined in 'b.go'

@
```

```go
// internal/suggest/testdata/test.0005/test.go.in (modified, replaced `//` with `/*`)
package main

...

/* methods SetC and SetD are defined in 'b.go'

@
```

Reply:
Thanks for investigating this. I'll merge the change and try to see if I can address this issue.

Reply:
Sounds good, thanks for the review! :smiley: